lightning/config/global.go

```diff
@@ -197,6 +197,9 @@ func LoadGlobalConfig(args []string, extraFlags func(*flag.FlagSet)) (*GlobalCon
 	if *logFilePath != "" {
 		cfg.App.Config.File = *logFilePath
 	}
+	if cfg.App.Config.File == "" {
```

Comment:
```suggestion
	else {
```

Reply:
This else is not ok, because toml config may set the log config path